public/pages/about-me.html

```diff
@@ -0,0 +1,105 @@
+                <img class="pointer" src="../resources/humburger.svg" id="humburger" alt="nav_menu">
+                <ul class="hidden-if-small">
+                    <li><a class="link" data-replace="About" href='./about-me.html'><span class="link-text">About</span></a></li>
+                    <li><a class="link main-program-link" data-replace="Program" href='./index.html#main-program'><span class="link-text">Program</span></a></li>
+                    <li><a class="link" data-replace="Join" href='#'><span class="link-text">Join</span></a></li>
+                    <li><a class="link" data-replace="Sponsor" href='./index.html#footer'><span class="link-text">Sponsor</span></a></li>
+                    <li><a class="link" data-replace="News" href='#'><span class="link-text">News</span></a></li>
+    
+                </ul>
+                <button type="button" class="hidden-if-small">
+                    <span>Campaign</span>
+                </button>
+            </nav>
+            <div class="hidden-if-large mobile-menu">
+                <div class="mobile-menu-top-section">
+                    <a href="./index.html"><img class="logo" src="../resources/kpop_black.png" alt="Logo"></a>
+                    <img src="../resources/cross.svg" width="30" class="pointer mobile-menu-img">
+                </div>
+                <ul class="mobile-menu-ul">
+                <li><a class="mobile-menu-ul-li-a link" data-replace="About" href='./about-me.html'><span class="link-text">About</span></a></li>
+                <li><a class="mobile-menu-ul-li-a link main-program-link" data-replace="Program" href='#main-program'><span class="link-text">Program</span></a></li>
+                <li><a class="mobile-menu-ul-li-a link" data-replace="Join" href='#'><span class="link-text">Join</span></a></li>
+                <li><a class="mobile-menu-ul-li-a link" data-replace="Sponsor" href='#footer'><span class="link-text">Sponsor</span></a></li>
+                <li><a class="mobile-menu-ul-li-a link" data-replace="News" href='#'><span class="link-text">News</span></a></li>
+                </ul>
+            </div>
+        </header>
+    <section class="about">
+        <h1 class="about-text-1"><span class="about-text-1-span hidden-if-large">"Hello! Sharing Fans"</span><span class="main-text">KPOP Conference 2022</span></h1>
+        <p class="about-text-2">The Korea Remixed K-pop Conference explores the historic, cultural, and political significance of South Korea&apos;s signature music. The conference will feature panels with IU experts, visiting scholars, and IU graduate and undergraduate students.</p>
+        <div class="about-contact">
+            <span class=".about-contact-span">Please Contact us per Email for any further questions</span>
+            <span class=".about-contact-span">about KPOP conference 2022!!</span>
+            <span class=".about-contact-span">ccsumit2015@cckorea.org</span>
+        </div>
+    </section>
```

Comment:
- The color of the page header is different from the design template. Please ensure you fix this 😄 . Remember it is essential that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project 👇:
<img width="957" alt="about capstone" src="https://user-images.githubusercontent.com/73607512/196648140-374684aa-80fe-4a02-8261-e6664b4dfe41.PNG">